products/BellHybrid/apps/application-bell-meditation-timer/presenter/MeditationProgressPresenter.hpp

```diff
@@ -79,11 +81,14 @@ namespace app::meditation
         void onProgressFinished();
         void onIntervalReached();
 
+        void addMeditationEntry(std::chrono::minutes elapsed);
+
       public:
         MeditationProgressPresenter(app::ApplicationCommon *app,
                                     settings::Settings *settings,
                                     std::unique_ptr<AbstractTimeModel> timeModel,
-                                    models::ChimeInterval &chimeIntervalModel);
+                                    models::ChimeInterval &chimeIntervalModel,
+                                    models::Statistics &statisticsModel);
```

Comment:
models::ChimeInterval &chimeIntervalModel,
models::Statistics &statisticsModel

I would add const here but its stylistic, without is also ok

Reply:
Adding `const` requires also adding `const` to  `models::ChimeInterval &chimeIntervalModel` and `models::Statistics &statisticsModel` which in this case is not possible :/